Shaders/TiltShift.fx

```diff
@@ -1,47 +1,56 @@
-// Based on kingeric1992's TiltShift effect
+/* 
+Tilt-Shift PS (c) 2018 Jacob Maximilian Fober, 
+(based on TiltShift effect (c) 2016 kingeric1992)
+
+This work is licensed under the Creative Commons 
+Attribution-ShareAlike 4.0 International License. 
+To view a copy of this license, visit 
+http://creativecommons.org/licenses/by-sa/4.0/.
+*/
 
 uniform bool Line <
 	ui_label = "Show Center Line";
 > = false;
 
-uniform float Axis <
+uniform int Axis <
+	ui_label = "Angle";
 	ui_type = "drag";
-	ui_min = -90; ui_max = 90; ui_step = 1;
-> = 0.0;
+	ui_min = -89; ui_max = 90; ui_step = 1;
+> = 0;
+
 uniform float Offset <
 	ui_type = "drag";
-	ui_min = -5; ui_max = 5;
-> = 0.0;
+	ui_min = -1.41; ui_max = 1.41; ui_step = 0.01;
+> = 0.05;
 
 uniform float BlurCurve <
+	ui_label = "Blur Curve";
 	ui_type = "drag";
-	ui_min = 0; ui_max = 10;
+	ui_min = 1.0; ui_max = 5.0; ui_step = 0.01;
 	ui_label = "Blur Curve";
 > = 1.0;
 uniform float BlurMultiplier <
+	ui_label = "Blur Multiplier";
 	ui_type = "drag";
-	ui_min = 0; ui_max = 100;
+	ui_min = 0.0; ui_max = 100.0; ui_step = 0.2;
 	ui_label = "Blur Multiplier";
-> = 10.0;
+> = 6.0;
```

Comment:
Do all these values behave the same way visually as they did with the old effect?

Reply:
The old effect did Gaussian blur also on alpha mask in the second pass, thus creating stronger falloff, but with blocky artifacts. I tried to match all values visual impact more or less with the original.
Also adding smaller steps, as previously one step moved the tilt around 1/8 of a screen.
Rotation angle now behaves more intuitively. Rotates at the center, not off-screen as previously.
By default tilt-line is slightly above the center, where before it was almost off-screen.
Also before high values created strong visible steps.